tests/test_cli.py

```diff
@@ -100,35 +100,11 @@ def test_returns_tabularised_representation_of_devices(self):
         )
         self.assertEqual(output, expected_output)
 
-    def test_displays_unknown_values(self):
```

Comment:
Did you not find this test useful?

Reply:
We don't want the unknown values to appear any more right?

Reply:
"UNKNOWN" value is used for missing serial port here, unless you've agreed on not displaying it offline, which I'm not aware of.

Reply:
Ah, right, sure I'll reinstate this test in that case.

Reply:
Test has returned in 7509007